README.md

```diff
@@ -55,8 +55,63 @@ Our Developer Success Team can assist with answering any questions as well as pr
 
 [Forums, Support and Knowledge Base](https://community.playfab.com/hc/en-us)
 
+6. Example Usage (Template):
+----
+```
+function VanillaPFTemplate(bar)
+{
+    function PFTemplate(bar)
+    {
+        console.log("@ PFTemplate..");
+
+        var req =
+        {
+            // TODO: Replace "fu" with your own request JSON KVP's
+            "fu": bar
+        };
+
+        // TODO: Uncomment below and replace FuBar with 
+        // a proper PlayFabClientSDK.function(request, callback)
```

Comment:
I would prefer a fully functional example which makes a real, specific api call, such as: https://api.playfab.com/Documentation/Client/method/LoginWithCustomID
It would be great if another PlayFab developer could copy paste this and just use it as their first API call.